willow/plugins/pillow.py

```diff
@@ -63,6 +64,33 @@ def resize(self, size):
     def crop(self, rect):
         return PillowImage(self.image.crop(rect))
 
+    @Image.operation
+    def rotate(self, angle=90):
+        """
+        Accept a multiple of 90 to pass to the underlying Pillow function
+        to rotate the image.
+        """
+
+        Image = _PIL_Image()
+        ORIENTATION_TO_TRANSPOSE = {
+            90: Image.ROTATE_90,
+            180: Image.ROTATE_180,
+            270: Image.ROTATE_270
+        }
+        transpose_code = ORIENTATION_TO_TRANSPOSE.get(angle)
```

Comment:
Might be nice to add ``% 360`` to the angle here

Reply:
Ah.

I left 360 out, as if you rotate an image 360 degrees, you rotate it _all_ the way around, and it ends up looking exactly the same as before.

That's my thinking at least - @simo97 's originally had a check if the angle was greater than 360, or less than -360, and you're right it might be worth covering the other angles someone might conceivably add.

Is there a use case you had in mind here, where you can imagine deliberately rotating an image 360 degrees?

Reply:
I actually meant that we should perform a modulo 360 operation on the angle, so that an input of 450 would be translated to 90. This would also have the effect of converting 360, 720 and -360 to 0 so these can all be handled generically.

> Is there a use case you had in mind here, where you can imagine deliberately rotating an image 360 degrees?

My thinking is that it's possible someone would add two angles together (or hit the rotate 90 button more than 3 times) and the developer forget to add ``% 360`` into their code themselves.

Reply:
Ah, _now_ I see where you're coming from - thanks for clearing that up.

I've pushed an update and a few tests for the cases you described 👍